src/Component.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ProcessorOrthogonal;
+
+use Keboola\Component\BaseComponent;
+use Keboola\Component\UserException;
+use Keboola\Csv\CsvFile;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+use Symfony\Component\Serializer\Encoder\JsonDecode;
+use Symfony\Component\Serializer\Encoder\JsonEncode;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+
+class Component extends BaseComponent
+{
+    /**
+     * @var JsonEncode
+     */
+    private $jsonEncode;
```

Comment:
Proč jsou encodery a decodery private fieldy? Jsou použitý jednou a instancoval bych je klidně až při použití. Má to nějakou výhodu?

Reply:
protoze je instancujes v cyklu, nema to zadnou vyhodu krome toho ze se udelaji jednou i kdyz jsou pouzity vickrat